watertap/costing/units/electrodialysis.py

```diff
@@ -13,41 +13,23 @@
 import pyomo.environ as pyo
 from ..util import (
     register_costing_parameter_block,
+    cost_rectifier,
     make_capital_cost_var,
     make_fixed_operating_cost_var,
 )
 
 
 def build_electrodialysis_cost_param_block(blk):
 
-    blk.cem_membrane_cost = pyo.Var(
-        initialize=43,
-        doc="Cost of CEM membrane used in Electrodialysis ($/CEM/area)",
+    blk.membrane_capital_cost = pyo.Var(
```

Comment:
Does it make sense to just name this `stack_capital_cost`? Technically, the same can be said of RO, where we can name it something like `module_cost` although I think it is not as straightforward for ED as it is for RO.

Reply:
I agree, lest rename this to stack_capital_cost. This will later aid us if we want to split cost of stack into membranes/electrodes, as we can add it just as a constraint that calculates stack capital cost.

Reply:
Methods changed with new names.